tests/generic/mongocollection-aggregatecursor-007.phpt

```diff
@@ -38,6 +39,7 @@ for ($i = 0; $i < 103; $i++) {
     $collection->insert(array('article_id' => $i));
 }
 
+// This assumes that the server's batch size defaults to 101 for aggregate
```

Comment:
In `tests/generic`, tests 006 and 008 for `aggregateCursor()` do test getmore requests with explicit batch sizes of 0 and 42, respectively.